packages/aws-cdk-lib/aws-cloudwatch/lib/graph.ts

```diff
@@ -364,6 +364,13 @@ export interface GraphWidgetProps extends MetricWidgetProps {
    */
   readonly liveData?: boolean;
 
+  /**
+   * Whether the graph should show labels on the chart. Only applicable for Pie charts.
+   *
+   * @default false
+   */
+  readonly displayLabelsOnChart?: boolean;
```

Comment:
Since this applies specifically to pie charts, the name can be more specific:
```suggestion
  readonly displayLabelsOnPieChart?: boolean;
```

Reply:
I think this depends on how confident we are that cloudwatch won't add more widgets in the future that have the same option to show labels on the graph. Ideally we would have something like a PieGraph class so we could have props that only make sense for pie charts. I think that would be too big of a change for this feature, though. 

I plan on just adding the validation and extra unit test. Please let me know if you disagree, and if we should still move forward with the rename.

Reply:
> I think this depends on how confident we are that cloudwatch won't add more widgets in the future that have the same option to show labels on the graph

I did think about, but I saw the comment above stating that it is for Pie charts only, which is what prompted the suggestion.

I think it's fine to keep the current naming, but let's update the JSdoc to clarify that this only works for Pie charts _for now_.